kubeadm v1.15 blog post

luxas opened the pull request:
cc @kubernetes/sig-cluster-lifecycle @castrojo 

FYI, I have checked "Allow edits from maintainers". Would appreciate grammar corrections, formatting fixes etc.
Thanks!

DanyC97 commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+to quickly and easily bootstrap minimum viable clusters that are fully compliant with 
+[Certified Kubernetes](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md) guidelines.
+It’s been under active development by [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)
+since 2016 and we graduated it from beta to
```
maybe we can rephrase it to `and graduated from beta to`

DanyC97 commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+[Certified Kubernetes](https://github.com/cncf/k8s-conformance/blob/master/terms-conditions/Certified_Kubernetes_Terms.md) guidelines.
+It’s been under active development by [SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle)
+since 2016 and we graduated it from beta to
+[stable and generally available (GA) in end of 2018](https://kubernetes.io/blog/2018/12/04 production-ready-kubernetes-cluster-creation-with-kubeadm/).
```
maybe `generally available (GA) at the end of 2018`

DanyC97 commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+The core of the kubeadm interface is quite simple: new control plane nodes are created by running 
+<strong><code>kubeadm init</code></strong>, worker nodes are joined to the control plane by running
+<strong><code>kubeadm join</code></strong>. Also included are common utilities for managing already bootstrapped
+clusters, such as control plane upgrades and token and certificate renewal.
```
maybe `control plane upgrades, token and certificate renewal` ?

DanyC97 commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+
+*   Infrastructure provisioning
+*   Third-party networking
+*   Non-critical add-ons for e.g. monitoring, logging, and visualization
```
maybe remove `for`

DanyC97 commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+
+### 2019 plans
+
+We are focusing our efforts this year around graduating the configuration file format to GA (`kubeadm.k8s.io/v1`)`, graduating this super-easy High Availability flow to stable, and providing better tools around rotating certificates needed for running the cluster automatically.
```
nit: `this year` can be taken out since the header is 2019 plans

sftim commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+You can argue that there are hardly two Kubernetes clusters that are configured equally, and hence there is a need to customize how the cluster is set up depending on the environment. One way of configuring a component is via flags. However, this has some scalability limitations:
+
+
+* **Hard to maintain.** When $component’s flag set grows over 30+ flags, configuring it becomes really painful
```
```suggestion
* **Hard to maintain.** When a component’s flag set grows over 30+ flags, configuring it becomes really painful.
```

sftim commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+
+Some handy links if you want to start contribute:
+
+*   We have a recorded [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
```
```suggestion
*   You can watch the SIG Cluster Lifecycle [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
```

sftim commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+*   We have a recorded [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
+*   Look out for “good first issue”, “help wanted” and “sig/cluster-lifecycle” labeled issues in our repositories 
+    (e.g. [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm))
+*   Join **#sig-cluster-lifecycle**, **#kubeadm**, **#cluster-api**, **#minikube**, **#kind**, etc. in Slack
```
Which Slack organisation? (I assume it's the main Kubernetes Slack).
Is it worth linking there?

sftim commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+# Automated High Availability in kubeadm v1.15: Batteries Included But Swappable
```
This blog post needs usual front matter. Have a look at other blog posts for an example.

sftim commented on content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md:
```diff
@@ -0,0 +1,202 @@
+This release wouldn’t have been possible without the help of the great people that have been contributing to SIG Cluster Lifecycle and kubeadm. We would like to thank all the kubeadm contributors and companies making it possible for their developers to work on Kubernetes!
+
+
+## Written by:
```
Usually, Kubernetes blog posts list the author(s) just below the front matter.

castrojo commented:
Ok I've addressed all the review comments, updated the logo, and added the front matter, PTAL.

luxas commented:
I also updated some minor things on it now (e.g. addressed the author and subproject owners attributions)
This is ready to merge IMO

@timothysc want to sign off?

luxas commented:
/assign @kbarnard10 @timothysc

castrojo commented:
/lgtm

stealthybox commented:
> During the course of the year, we’re looking forward to graduate the schema to General Availability v1.` If you’re interested in this effort, you can also join WG Component Standard.

Nice :ok_hand: 

/lgtm

kbarnard10 commented:
/approve

luxas commented:
Just merge whenever you're ready. Thanks!

kbarnard10 commented:
/lgtm
